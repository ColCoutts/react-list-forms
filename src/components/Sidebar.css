.Sidebar {
  display: flex;
  flex-direction: column;
  justify-content: space-around;
  width: 25vw;
  height: 100vh;
  padding: 1rem;

  background-color: black;
  color: white;
}
