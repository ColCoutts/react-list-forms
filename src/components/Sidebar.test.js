import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from './Sidebar';

describe('sidebar component', () => {
  it('renders a sidebar', () => {
    const wrapper = shallow(
      <Sidebar>
        <a href="hi">Test</a>
      </Sidebar>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
