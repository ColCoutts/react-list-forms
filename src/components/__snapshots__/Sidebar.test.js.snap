// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sidebar component renders a sidebar 1`] = `
<nav
  className="Sidebar"
>
  <a
    href="hi"
  >
    Test
  </a>
</nav>
`;
